### ISSUE TITLE
Windows support

### ISSUE THREAD

**Issue by willothy** [willothy/moveline.nvim#4]
This plugin currently can't support Windows as the mlua crate cannot package a vendored luajit library on Windows. I'm working on a way to allow windows support by using the CC crate to compile the Luajit library as needed on Windows, so expect that soon :)

**Comment by sijirama** [comment]
Yeah this would be nice, i was about to open an issue for windows support too.

**Comment by highwaycoder** [comment]
I don't wanna be that guy, but is there any update on this? It just became a minor problem for me as I copied my init.lua across to a new PC (the only one in my horde that runs Windows) and I'm gonna really miss moveline when I'm working on this machine 😅

**Comment by willothy** [comment]
I will look into this again soon, sorry! In the mean time I would recommend using a similar plugin written in Lua.

FYI, I am considering archiving this plugin since it was originally written as a proof of concept / test for my library nvim-utils, which I have also archived in favor of just using `nvim-oxi`. I also do not think this is the right type of plugin to be written in Rust, since it's really only doing Vim things and is probably just made *slower* by the FFI.

I may also finish up the nvim-oxi port and move that to the main branch. I haven't decided yet. I'm glad people like the plugin though :)

I haven't tested these as I still use this plugin myself (I use the oxi branch), but here are some Lua-based alternatives:

- [nvim-gomove](https://github.com/booperlv/nvim-gomove)
- [move.nvim](https://github.com/hinell/move.nvim)
- [mini.move](https://github.com/echasnovski/mini.nvim/blob/main/readmes/mini-move.md)
